recipes/esys-escript/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set version = "5.5" %}
+
+package:
+    name: esys-escript
+    version: {{ version }}
+source:
+    - url: https://launchpad.net/escript-finley/3.0+/{{ version }}/+download/python-escript_{{ version }}.tar.gz
+      sha256: 78cb3c43a04426c4909c6f1ca1aa23cbec574d783fd438fcfd419931b7c63be8
+      folder: escript
+build:
+    skip: True  # [osx]
+    skip: True  # [win]
+    skip: True  # [py!=27]
```

if you want to, it can be just
```suggestion
    skip: True  # [not linux or py3k]
```